.flake8

```diff
@@ -1,2 +1,5 @@
 [flake8]
-max-line-length = 120
\ No newline at end of file
+max-line-length = 120
+
+# Prevents __init__.py from erroring for unused imports
+per-file-ignores = __init__.py:F401
```

Comment:
401 is unused imports. This makes sure our __init__.py doesn't error when it shouldn't.